tbxforms/layout/fields.py

```diff
@@ -72,7 +72,12 @@ class Field(crispy_forms_layout.LayoutObject):
     template = "%s/field.html"
 
     @classmethod
-    def checkbox(cls, field, small=False, **kwargs):
+    def checkbox(
```

Comment:
will need to be highlighted in upgrade considerations

Reply:
Would you point this out? It has never worked

Reply:
The comment was meant to go on https://github.com/torchbox/tbxforms/pull/51/files#diff-bbecf68572c33c583f20a5c9ebe61e8fd644ee8f20d2764735f49b65f9a2fe38L169 (i.e. anywhere leged change to label). I would point it out if only for people to update their signatures and to know what the correct solution is